playbooks/tests/tasks/rally/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+source /root/stackrc
+
+BUILD_TAG=${1:-master}
+RALLY=~/rally/bin/rally
+RALLY_INSTALL_URL=https://raw.githubusercontent.com/openstack/rally/master/install_rally.sh
+RALLY_FILE=bbc-cloud-validate
+RALLY_TEST_IMAGE="cirros"
+RALLY_TEST_NET_ID=$(openstack network show internal -c id -f value)
+RALLY_TEST_VCPU_LIMIT=4
+
+read -r -d '' RALLY_SED_STRING <<EOM
+s/@@NET_ID@@/${RALLY_TEST_NET_ID}/g;
+s/@@IMAGE_NAME@@/${RALLY_TEST_IMAGE}/g;
+s/@@VCPU_LIMIT@@/${RALLY_TEST_VCPU_LIMIT}/g
+EOM
```

Comment:
Why is this done instead of using Rally's native way to template the file and supply data at run time? https://rally.readthedocs.io/en/latest/cli/cli_reference.html#task-start-taskargs